build/gulp/templates/playground-angular-declaration-application.jst

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable spellcheck/spell-checker */
```

Comment:
Why do you have to disable spellcheck for the whole file? Maybe some word should be added to the dictionary or ignored locally?